backend/src/main/java/fk/prof/backend/http/LeaderHttpVerticle.java

```diff
@@ -181,30 +227,70 @@ private void handleGetWork(RoutingContext context) {
     }
   }
 
-  private void handlePostPolicy(RoutingContext context) {
+  private void handleGetPolicy(RoutingContext context) {
     try {
-      String appId = context.request().getParam("appId");
-      String clusterId = context.request().getParam("clusterId");
-      String procName = context.request().getParam("procName");
-
-      // for now expecting a json payload
-      String payload = context.getBodyAsString("utf-8");
-
-      BackendDTO.RecordingPolicy.Builder builder = BackendDTO.RecordingPolicy.newBuilder();
-      JsonFormat.parser().merge(payload, builder);
-
-      BackendDTO.RecordingPolicy policy = builder.build();
-      Recorder.ProcessGroup pg = Recorder.ProcessGroup.newBuilder().setAppId(appId).setCluster(clusterId).setProcName(procName).build();
+      Recorder.ProcessGroup pG = parseProcessGroup(context);
+      PolicyDTO.VersionedPolicyDetails versionedPolicyDetails = policyStore.getVersionedPolicy(pG);
+      if (versionedPolicyDetails == null) {
+        context.response().setStatusCode(400).end("Policy not found for ProcessGroup " + RecorderProtoUtil.processGroupCompactRepr(pG));
```

Comment:
But having exact variable names in the logs can help maintainers search easily. if yes, then we can go for something like processGroup for vars and CamelCase for class names?